src/Input.tsx

```diff
@@ -16,6 +16,8 @@ interface Props {
   style?: React.CSSProperties;
   // optional default value
   defaultValue?: string;
+  // optional autoFocus
+  autoFocus?: boolean;
```

Comment:
if this should be a `boolean`, then we need to provide a default prop instead of having `undefined` applied on the rendered input.